features/testbot/zero-speed-updates.feature

```diff
@@ -111,8 +111,8 @@ Feature: Check zero speed updates
             """
 
         When I route I should get
-          | from | to | bearings | code    |
-          |    1 |  2 | 270 270  | NoRoute |
```

Comment:
The error changed because in the `bicycle` profile oneways are actually traversable in both directions (if you push your bike), right?